style/tables.css

```diff
@@ -13,6 +13,13 @@
   box-sizing: border-box;
   position: relative;
 }
+
+.ProseMirror td:not([colwidth]):not(.column-resize-dragging),
+.ProseMirror th:not([colwidth]):not(.column-resize-dragging) {
```

Comment:
```suggestion
.ProseMirror td:not([data-colwidth]):not(.column-resize-dragging),
.ProseMirror th:not([data-colwidth]):not(.column-resize-dragging) {
```

Reply:
Why do you prefer data-colwidth?

I think colwidth is safer, because not everyone might set data-colwidth in the toDOM method, while colwidth is always necessary I think?

Reply:
ProseMirror-tables doesn't set a `colwidth` attribute to DOM. It only set `data-colwidth`. Right?

Reply:
If someone else use a different `toDOM` method than the default one in ProseMirror-tables, they can adjust the CSS too.

Reply:
you're right, good spot. Fixed